dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -84,20 +84,18 @@ def testing():
         allow_extra_args=True,
     ),
 )
-@option_verbose
-@option_dry_run
 @option_python
-@option_github_repository
 @option_image_tag_for_running
 @option_image_name
+@option_github_repository
+@option_verbose
+@option_dry_run
 @click.argument("extra_pytest_args", nargs=-1, type=click.UNPROCESSED)
 def docker_compose_tests(
-    verbose: bool,
-    dry_run: bool,
     python: str,
-    github_repository: str,
     image_name: str,
     image_tag: str | None,
+    github_repository: str,
```

Comment:
It will use it. github_repository is not as common and used to pass to buildParams to construct Image name, so I decided to left it as "regular param' (but it cross my mind to convert it to "static" param.